data/json/items/generic.json

```diff
@@ -1167,6 +1167,19 @@
     "volume": 1,
     "to_hit": 2
   },
+  {
+    "type": "GENERIC",
+    "id": "motor_micro",
+    "symbol": ",",
+    "color": "light_gray",
+    "name": "micro motor",
+    "category": "spare_parts",
+    "description": "A very small electric motor like those used in RC cars.  Useful in lots of electronics recipes.",
+    "price": 8000,
```

Comment:
Way too much for a RC-car motor.

Reply:
The price is too much ? It's the same as the antenna, what would be a more reasonnable one ?